### ISSUE TITLE
Enable Track by Name tracking in CrazyEgg 

### ISSUE THREAD

**Issue by nickpistone** [GSA/px-benefit-finder#1002]
### Description

Since it's a single-page app, the heatmap can only see the Start page. We need to use custom tracking such as Track by Name to get snapshots of each virtual page in the benefit finder tool. We need to collaborate with David Kaufmann and Nicole Brennan to align on how to implement this. The link below explains the setup. We also need to install something called Page Camera and we may need to set up a call with CrazyEgg support as well.

https://support.crazyegg.com/hc/en-us/articles/360054584414-Track-by-Name

**Comment by nickpistone** [comment]
Scott and I created a test snapshot this morning for DOLO about the applicant (virtual page 1). I'll be able to verify if it worked by tomorrow. If it works, then we can make additional snapshots for each page of the form for all life experiences and languages. However, there is a limit on how many pageviews we can use, so we might not be able to collect much data in March since USAGov is running a big A/B test that has already used most of our pageviews for the month.

**Comment by nickpistone** [comment]
<img width="980" alt="Screenshot 2024-03-06 at 5 44 49 PM" src="https://github.com/GSA/px-benefit-finder/assets/121254111/cb9a58b4-2e10-4f90-bf01-3b58630a2087">

I just confirmed that the Track by Name configuration worked. I had to refresh the snapshot using the Page Camera plugin. But it is now working as seen in this screenshot. We can now repeat this for the other life experiences, but we will need to prioritize so we don't make too many snapshots.
